puppet/zulip/files/nginx/zulip-include-frontend/app

```diff
@@ -76,6 +76,15 @@ location ~ ^/internal/tornado/(\d+)(/.*)$ {
     include /etc/nginx/zulip-include/proxy_longpolling;
 }
 
+location /api/v1/tus {
+    include /etc/nginx/zulip-include/api_headers;
+    include /etc/nginx/zulip-include/proxy;
+    client_max_body_size    0;
+    proxy_request_buffering off;
+    proxy_buffering         off;
+    proxy_pass              http://tusd;
+}
```

Comment:
Maybe worth some comments documenting those settings; `max_body_size` of 0 will seem weird to a reader not familiar with what this is doing.